### ISSUE TITLE
Options Panel

### ISSUE THREAD

**Issue by tobodi** [WaldiPL/webpageScanner#14]
I decide to try your addon latest version on my Firefox ESR 52.4.0 Windows 10.

Unfortunately the sidebar is active only from Firefox 54+. Like to many people, I will stick with the 52 ESR version for a long time with this latest Firefox changes and is not nice if this option is missing. 

The second thing, which I saw is when I access the option panel for the add on. When I click on the option button for the add on, I am not able to see anything properly. Please, see the screenshot.

Hopefully, you can make compatible this add on properly from Firefox ESR 52, because too many people like me will stick with this version.

**Comment by tobodi** [comment]
The screen shot is:
![websc](https://user-images.githubusercontent.com/32428674/31052732-0cb332b2-a685-11e7-8ac4-a940e0dac1f7.jpg)

**Comment by WaldiPL** [comment]
Yes, the sidebar is available from version 54. You can open the list in a tab or on a popup (there may be problems).

I can not reproduce the bug with options.
Does the text in other places display correctly? (eg popup, button)
What language do you use? (operating system language)

You can help locate the problem.
1. Open extension options
2. Open the browser console (Ctrl+Shift+J)
3. Clear the console (trash icon)
4. Refresh the options.
5. Make a screenshot of the console and upload it here.

**Comment by tobodi** [comment]
Do you think you can make the sidebar to work in Firefox 52 ESR?

The screenshot  of the console is:

![untitled2](https://user-images.githubusercontent.com/32428674/31058492-620bdcb6-a6ec-11e7-8dac-e1d2ca8bd2b3.jpg)

**Comment by WaldiPL** [comment]
There is no way to create a sidebar in Firefox 52.

The problem can be triggered by conflict with another extension. You are using some legacy extensions? (non-webextension)
Try opening options in a new tab. Copy the link below to the address bar
`moz-extension://ce7a901d-4f86-40b2-88d6-6feaf21555a3/options.html`

**Comment by tobodi** [comment]
The problem is cause by the extension OmniSidebar.

Thank you for the confirmation for the sidebar, which is missing in Firefox 52 ESR.
